src/Runner/Runner.php

```diff
@@ -134,8 +134,16 @@ public function fix()
             Tokens::clearCache();
 
             if ($fixInfo) {
-                $name = $this->directory->getRelativePathTo($file);
-                $changed[$name] = $fixInfo;
+                $path = $file->getPathname();
+
+                /* workaround for Symfony/Finder setting doubled slash for root folders in
+                SplFileInfo. See `RecursiveDirectoryIterator::current()`. */
```

Comment:
Have not found officialy reported one, but also not seen all. Made a fork already, planning to make a PR there with a fix - when open will link it. I should have some time tomorrow so hopefully will grab this on.